.env.example

```diff
@@ -66,8 +65,9 @@ BADGE_TOKEN_BASE={bundle exec rake secret} # optional
 # report_process_stats:60
 # periodical_deploy:86400
 # cancel_stalled_builds:3600
+# repo_provider_status:60
```

Comment:
Add a note that this depends on the `samson_github` plugin (or a plugin in general with the correct hook implemented) being enabled?